MdeModulePkg/Bus/Pci/PciBusDxe/PciOptionRomSupport.c

```diff
@@ -243,6 +243,7 @@ GetOpRomInfo (
   UINT8                           Bus;
   UINT8                           Device;
   UINT8                           Function;
+  UINT8                           OriginalValue;

```

Comment:
Define OriginalValue as a local variable ?

Reply:
It is already defined as local variable. As the code line is collapsed,  it is mistreated as function parameter.
After expanding the code line,  we know that it is the local variable.